app/views/landing.erb

```diff
@@ -1,6 +1,56 @@
 
-<top-border></top-border>
+<script src="/javascripts/landing.js" type="text/javascript"></script>
+<script src="/javascripts/tab-forms.js" type="text/javascript"></script>
+<div id="loginForm" class="overlay">
+  <a href="javascript:void(0)" class="closebtn" onclick="closeForms()">&times;</a>
```

Comment:
Hah I've never actually used this trick.. Can do `href="#"` instead, but I think that might actually add the '#' to the URL...

Another alternative (and arguably a [more semantic one](https://davidwalsh.name/html5-buttons)) would be to use a `<button>` instead of an `<a>` and just assigning it a class that clears/changes the default button styling.

(That being said, we use clickable/non-href `<a>` tags everywhere, so it's really just preference)